### PR TITLE
Pmdconfig

### PR DESCRIPTION
**Pull request by rutgervandenberg**
Addresses #106

### CODE REVIEW

**Comment by joswinter** [comment]
Good job, looks good!

File: pmd.ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<ruleset name="Custom ruleset"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+  <description>
+  This ruleset checks my code for bad stuff
```
Comment (jorenham) [inline pmd.ruleset.xml:7]: Description needs an update.

**Comment by albert-smit** [comment]
looks good, but name and description could be better

**Comment by alangerak** [comment]
Looks good

**Pull request merged**